README.md

```diff
@@ -8,6 +8,7 @@ run Frontend : npm start
 ## styling 
 sass module 
 styled component 
+classname
```

Comment:
Now I can comment on this line of code but no other files, I could review and add comments only on the changed files in the PR, got it?

Reply:
yes, already added all the main files in the PR 
thank you in advance